platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/SystemPropertiesAdminTest.java

```diff
@@ -158,7 +158,7 @@ public void testWriteSystemProperties() throws Exception {
     Properties sysProps = new Properties();
     try (FileReader sysPropsReader = new FileReader(systemPropsFile)) {
       sysProps.load(sysPropsReader);
-      assertThat(sysProps.size(), is(3));
```

Comment:
✏️ Maybe also verify that the internal properties are not overwritten by `writeSystemProperties`